test-data/unit/check-custom-plugin.test

```diff
@@ -991,3 +991,20 @@ class Cls:
 [file mypy.ini]
 \[mypy]
 plugins=<ROOT>/test-data/unit/plugins/class_attr_hook.py
+
+[case testAddClassMethodPlugin]
+# flags: --config-file tmp/mypy.ini
+class BaseAddMethod: pass
+
+class MyClass(BaseAddMethod):
+    pass
+
+my_class = MyClass()
+reveal_type(MyClass.foo_classmethod())  # N: Revealed type is "None"
```

Comment:
Let's add `reveal_type(MyClass.foo_classmethod)` and `reveal_type(MyClass.foo_staticmethod)`.

This check would ensure that our signatures are correct.

Reply:
I can add the reveal_types, but I honestly would still leave the other reveal_types that check calling the function works.